apps/nextra/pages/en/build/create-aptos-dapp/templates/digital-asset.mdx

```diff
@@ -98,77 +129,114 @@ Metadata files are `json` files that hold the Collection data and each of the NF
       <FileTree.File name="2.jpeg" />
       <FileTree.File name="3.jpeg" />
     </FileTree.Folder>
-    <FileTree.Folder name="metadatas">
+    <FileTree.Folder name="metadata">
       <FileTree.File name="1.json" />
       <FileTree.File name="2.json" />
       <FileTree.File name="3.json" />
     </FileTree.Folder>
   </FileTree.Folder>
 </FileTree>
+</Steps>
 
-<Callout type="info">
-  Make sure all image files have the same file extension. If some images will
-  have `jpeg` extension and some `jpg` extension, the upload will fail.
-</Callout>
+## Publish the Move Contract
 
-{ /* TODO: change ERC-1155 once finish rewriting the NFT standards page */ }
+<Steps>
+### Prepare an account, or create one with an Aptos wallet like [Petra](https://petra.app/docs/use#create-a-new-account).
+You will need this to run the contract to create a collection. Make sure to make the account on the same network as your template project.
+
+### Use the below command to create the contract to mint your collection.
 
-We recommend the developers follow the [ERC-1155 off-chain data](https://eips.ethereum.org/EIPS/eip-1155) schema to format their JSON files. One thing to note, since the image field will be updated with the Arweave link after this step, feel free to leave it empty.
+```bash npm2yarn
+npm run move:init
+```
 
-Example of `collection.json` file
+The CLI will prompt you to provide the secret for your wallet in order for the tool to make the collection on your behalf.
 
-```js filename="collection.json"
-{
-  "name": "Aptos NFTs",                         // Name of the collection.
-  "description": "My NFT Collection on Aptos.", // Description of the collection
-  "image": "to_fill_after_upload",              // This is the URL to the image of the collection, we will assign Arweave link so feel free to leave it empty
-  "external_url": "https://your_project_url.io" // URL to an external website where the user can also view the image.
-}
+This will create the contract that handles minting your collection, as well as setting the `VITE_MODULE_ADDRESS` in the `.env` file to indicate which account will own any collections that get minted.
```

Comment:
```suggestion
This will create the contract for minting your collection, as well as setting the `VITE_MODULE_ADDRESS` in the `.env` file to set which account will own the minted collections.
```